quickstart/maven.markdown

```diff
@@ -238,7 +238,7 @@ For example when running on OpenJDK 7 the it is sometimes necessary to disable t
 
 ```xml
 <jvmArgs>
-    <value>-XX:-UseSplitVerifier</value>
+    <jvmArg>-XX:-UseSplitVerifier</jvmArg>
```

Comment:
yes following https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Mapping_Lists this is the correct way to specify individual entries. But Maven isn't that strict here, <value> does work either.

Nevertheless: :thumbsup: We should base the examples on Maven's recommendations